stand/loader.mk

```diff
@@ -167,13 +167,18 @@ CFLAGS+=	-DELF_VERBOSE
 .error Define HELP_FILENAME before including loader.mk
 .endif
 
+CFLAGS+=	-DHELP_FILENAME=\"${HELP_FILENAME}\"
+
+# If we are making the ia32 loader, we dont need to make another
+# help file.
```

Comment:
What's wrong with just setting INSTALL_LOADER_HELP_FILE in the including Makefile?